.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -246,6 +246,7 @@ steps:
       -e OS_NAME="Windows" \
       -e SKU_NAME=${SKU_NAME} \
       -e OFFER_NAME="Windows" \
+      -e MODE=$(MODE) \
```

Comment:
It seems like that https://github.com/Azure/AgentBaker/commit/16c95f214e694a719c989d637338e043a39b5eb1#diff-058b750b8d1a15973e955b1bb546b9c742e9185a4bf4f286c7ff827d071494caR70 blocked windows vhd build. cc @cameronmeissner